pkg/resourcemonitor/resourcemonitor_test.go

```diff
@@ -474,13 +474,15 @@ func TestResourcesScan(t *testing.T) {
 					Resources: topologyv1alpha1.ResourceInfoList{
 						topologyv1alpha1.ResourceInfo{
 							Name:        "cpu",
-							Allocatable: intstr.FromString("12"),
-							Capacity:    intstr.FromString("12"),
+							Available:   resource.MustParse("12"),
+							Allocatable: resource.MustParse("12"),
+							Capacity:    resource.MustParse("12"),
 						},
 						topologyv1alpha1.ResourceInfo{
 							Name:        "fake.io/net",
-							Allocatable: intstr.FromString("4"),
-							Capacity:    intstr.FromString("4"),
+							Available:   resource.MustParse("12"),
```

Comment:
This looks like a mistake to me: available should be equal to `4` here as well. Also note that available will always be <= both allocatable and capacity.

Reply:
yes. copy paste mistake I just found out myself.